packages/app-page-builder/src/render/plugins/elements/button/Button.tsx

```diff
@@ -9,10 +10,26 @@ const Button = ({ element }: { element: PbElement }) => {
     const {
         responsiveDisplayMode: { displayMode }
     } = React.useContext<PageBuilderContextValue>(PageBuilderContext);
-    const { type = "default", icon = {}, link = {} } = element.data || {};
+    const { type = "default", icon = {}, action = {}, link = {} } = element.data || {};
     const { svg = null } = icon;
     const { position = "left" } = icon;
 
+    console.log("action: ", action.clickHandler);
+    console.log("plugin: ", plugins.byName(action.clickHandler));
+    const plugin = useMemo(() => plugins.byName(action.clickHandler), []);
```

Comment:
Hm... it seems this will only memoize the plugin that was initial returned from the `plugins.byName` call. What happens if user changes the handler? 

In other words, let's ensure the plugin is recalculated every time the name changes. So, instead of passing `[]` as the 2nd arg to`useMemo`, let's pass `[action.clickHandler]`.